ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -193,6 +203,11 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
     -> BlockQuery (ShelleyBlock proto era)
                   (SL.PState (EraCrypto era))
 
+  GetStakeSnapshot
```

Comment:
Should this be `GetStakeSnapshots`? Because the return type does include the mark/set/go triplicates...

Minor: this awkward mix of singulars and plurals would also be avoided by my other comment's suggestion of how to return a restricted `SnapShots crypto` as Jordan has originally suggested.